CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Pelion Edge 2.5.2
+
+- Fix compilation error related to `mbed-fcce`.
+- Pull in `opencontainer/runc` recipe to `meta-pelion-edge` to fix master to main rename build breakage.
+
```

Comment:
Think we've changed more than this.

Reply:
True, updated the CHANGELOG.md accordingly.